frontend/src/screens/wallet/send/Onchain.tsx

```diff
@@ -0,0 +1,360 @@
+          className="w-full md:w-fit"
+        >
+          Send
+        </LoadingButton>
+      </div>
+    </form>
+  );
+}
+
+function SwapForm({
+  address,
+  setSwap,
+}: {
+  address: string;
+  setSwap: React.Dispatch<React.SetStateAction<boolean>>;
+}) {
+  const navigate = useNavigate();
+  const { toast } = useToast();
+  const { data: balances } = useBalances();
+  const { data: swapFees } = useSwapFees("out");
+
+  const [amount, setAmount] = React.useState("");
+  const [isLoading, setLoading] = React.useState(false);
+
+  const onSubmit = async (event: React.FormEvent<HTMLFormElement>) => {
+    event.preventDefault();
+    try {
+      setLoading(true);
+      const swapOutResponse = await request<SwapResponse>("/api/swaps/out", {
+        method: "POST",
+        headers: {
+          "Content-Type": "application/json",
+        },
+        body: JSON.stringify({
+          swapAmount: +amount,
+          destination: address,
+          isSending: true,
+        }),
+      });
+      if (!swapOutResponse) {
+        throw new Error("Error swapping out");
+      }
+      navigate(`/wallet/swap/out/status/${swapOutResponse.swapId}`);
+      toast({ title: "Initiated swap" });
+    } catch (e) {
+      toast({
+        variant: "destructive",
+        title: "Failed to send payment",
+        description: "" + e,
+      });
+      console.error(e);
+    } finally {
+      setLoading(false);
+    }
+  };
+
+  if (!balances) {
+    return <Loading />;
+  }
+
+  return (
+    <form onSubmit={onSubmit} className="grid gap-6">
+      <div className="grid gap-2">
+        <Label htmlFor="amount">Amount</Label>
+        <Input
+          id="amount"
+          type="number"
+          value={amount}
+          placeholder="Amount in Satoshi..."
+          onChange={(e) => {
+            setAmount(e.target.value.trim());
+          }}
+          min={MIN_SWAP_AMOUNT}
+          max={Math.floor(balances.lightning.totalSpendable / 1000)}
+          required
+          autoFocus
+          className="[appearance:textfield] [&::-webkit-outer-spin-button]:appearance-none [&::-webkit-inner-spin-button]:appearance-none"
+          endAdornment={
+            <FormattedFiatAmount amount={Number(amount)} className="mr-2" />
+          }
+        />
+        <div className="grid gap-1">
+          <div className="flex justify-between text-muted-foreground text-xs sensitive slashed-zero">
+            <div>
+              Spending Balance:{" "}
+              {new Intl.NumberFormat().format(
+                Math.floor(balances.lightning.totalSpendable / 1000)
+              )}{" "}
+              sats
+            </div>
+            <FormattedFiatAmount
+              className="text-xs"
+              amount={Math.floor(balances.lightning.totalSpendable / 1000)}
+            />
+          </div>
+          <div className="flex justify-between text-muted-foreground text-xs sensitive slashed-zero">
+            <div>Minimum: 50000 sats</div>
```

Comment:
I noticed this is hard coded but in Boltz right now the minimum is 25,000. And maybe if fees increase then the minimum would be more than 50,000. Can we fetch these limits from Boltz?